DragonFruit.Six.API/Stats/SeasonalStats.cs

```diff
@@ -39,6 +48,31 @@ public class Season
         [JsonProperty("maxmmr")]
         public double MaxMMR { get; set; }
 
+        [JsonProperty("next_rank_mmr")]
+        public double NextRankMMR { get; set; }
+
+        [JsonProperty("previous_rank_mmr")]
+        public double PreviousRankMMR { get; set; }
+
+        [JsonProperty("skill_mean")]
+        public double SkillMean { get; set; }
+
+        [JsonProperty("skill_stdev")]
+        public double SkillStdev { get; set; } // TODO: Wasn't sure what to call this... Needs to be named more appropriately
+
+        [JsonProperty("last_match_result")]
+        public uint LastMatchResult { get; set; }
```

Comment:
I think they are win, loss and draw. Hard to confirm the draw. I can test this today.

Reply:
Is it even possible to draw on a ranked match? I would assume Win = 1 and loss = 0

Reply:
You are completely correct, it is not possible to draw on ranked. I had a bit of a brain fart there as it is possible to draw on custom match.

An `enum` works here.

Reply:
When entering a Uplay name that doesn't have any playtime, the match result returns 0. On my Uplay account it returns 2, although I do not know if that is a win or loss, seems I have played two games today and neither of them show up in my stats anywhere (my MMR has not even updated in game since my last game).

The `enum` will have to wait until we can check the result for win or loss. I believe default is 0, loss 1 and win 2.